help/opers/cmode

```diff
@@ -2,6 +2,8 @@ MODE <channel> <+|-><modes> [parameters]
 
 ? designates that the cmode is provided by an extension
 and may not be present on this server.
+
+* designates that the cmode is oper only.
```

Comment:
We don't need a new symbol for this. It's true for other modes too, like `L` and `P` and the goes into the comment about it.

Reply:
@glguy - Fixed below - let me know if any further corrections are suggested